modules/apps/commerce/commerce-frontend-js/src/main/resources/META-INF/resources/components/mini_cart/OrderButton.js

```diff
@@ -17,13 +17,17 @@ import React, {useContext} from 'react';
 
 import {liferayNavigate} from '../../utilities/index';
 import MiniCartContext from './MiniCartContext';
-import {WORKFLOW_STATUS_APPROVED} from './util/constants';
+import {
+	REVIEW_ORDER,
+	SUBMIT_ORDER,
+	WORKFLOW_STATUS_APPROVED,
+} from './util/constants';
 import {hasErrors} from './util/index';
 
 function OrderButton() {
-	const {actionURLs, cartState} = useContext(MiniCartContext),
-		{cartItems = []} = cartState,
-		{length: numberOfItems = 0} = cartItems,
+	const {actionURLs, cartState, labels} = useContext(MiniCartContext),
+		{cartItems} = cartState,
+		{length: numberOfItems = 0} = cartItems || {},
```

Comment:
That `{}` looks like it should be a `[]` — I'd expect an array intuitively, and other places in this PR we have `cartItems` that look to be arrays.

Reply:
As above, I agree in this case it's also more misleading for the object type.